scipy/stats/_multivariate.py

```diff
@@ -3892,8 +3873,16 @@ class multivariate_t_gen(multi_rv_generic):
 
     Examples
     --------
-    >>> import matplotlib.pyplot as plt
+    The object may be called (as a function) to fix the `loc`, `shape`,
+    `df`, and `allow_singular` parameters, returning a "frozen"
+    multivariate_t random variable:
+
     >>> from scipy.stats import multivariate_t
+    >>> rv = multivariate_t([1.0, -0.5], [[2.1, 0.3], [0.3, 1.5]], df=2)
+    >>> # Frozen object with the same methods but holding the given location,
+    >>> # scale, and degrees of freedom fixed.
```

Comment:
In all the other examples, this bit about freezing the distribution comes after the other example. This example below already freezes the distribution, though, so perhaps this comment is not needed. Really, all of these just need some commentary.

Reply:
comment removed.

Reply:
Oops, I guess I wasn't clear, but we'll leave it as is now.